pkgs/top-level/all-packages.nix

```
@@ -12831,6 +12831,8 @@ with pkgs;
 
   stylish-cabal = haskell.lib.compose.justStaticExecutables haskellPackages.stylish-cabal;
 
+  lhs2tex = haskellPackages.lhs2tex;
```

This will reduce the closure size of the package and actually give us a benefit in exposing it top level.

```suggestion
  lhs2tex = haskell.lib.compose.justStaticExecutables haskellPackages.lhs2tex;
```

---

I tried that and got the same content in the closure, presumably because it only builds a binary in the first place? Therefore I did not use this function, so that accessing it via top level and via haskellPackages yield the same derivation. But let me double check maybe…

---

Doublechecked:
```
~/build/nixpkgs $ nix-build -E 'with (import ./. {}); haskell.lib.compose.justStaticExecutables haskellPackages.lhs2tex'
/nix/store/b5hh19d2pd1qn6snpqlnazj4d0v3zp1i-lhs2tex-1.24
~/build/nixpkgs $ nix-build -E 'with (import ./. {}); haskellPackages.lhs2tex'
/nix/store/sq6djqzpys0h5rni3ipzya9gwc5pzqb0-lhs2tex-1.24
~/build/nixpkgs $ diff -Nur /nix/store/sq6djqzpys0h5rni3ipzya9gwc5pzqb0-lhs2tex-1.24 /nix/store/sq6djqzpys0h5rni3ipzya9gwc5pzqb0-lhs2tex-1.24
~/build/nixpkgs $ 
```